lnrpc/lightning.proto

```diff
@@ -3843,7 +3843,12 @@ message Invoice {
     Signals that the invoice should include blinded paths to hide the true
     identity of the recipient.
     */
-    BlindedPathConfig blinded_path_config = 29;
+    bool is_blinded = 29;
+
+    /*
+    Config values to use when creating blinded paths for this invoice.
+    */
+    BlindedPathConfig blinded_path_config = 30;
```

Comment:
note that this change in field number is fine if we get this into 0.18.3 (if it doesnt make that release then i'll change `is_blinded` to 30